scripts/platoon-launcher

```diff
@@ -38,28 +38,37 @@ if __name__ == '__main__':
     os.makedirs(logs_folder)
 
     print "### Launching experiment: {}".format(args.experiment_name)
-    process_list = {}
+    process_map = {}
 
-    process_list['Controller'] = launch_process(logs_folder, args.experiment_name, args.controller_args, "cpu", "controller")
+    p = launch_process(logs_folder, args.experiment_name, args.controller_args, "cpu", "controller")
+    process_map[p.pid] = ('Controller', p)
```

Comment:
We should check if the controller actually launched before starting the workers.

Imagine the case you have a controller already running on the same port you are trying to use, an old one or just somebody else controller. Then you would launch worker what would work on with the other controller which is bad.